game_api.txt

```diff
@@ -228,6 +230,9 @@ The doors mod allows modders to register custom doors and trapdoors.
 
 	description = "Trapdoor description",
 	inventory_image = "mod_trapdoor_inv.png",
+	closed = {} -- Nodebox for closed model
+	opened = {} -- Nodebox for opened model
```

Comment:
Maybe rename these to `nodebox_closed` and `nodebox_opened`.

Reply:
I'll keep that in mind. I might change it later.

Reply:
I agree with DS-Minetest.

Reply:
Will do.